README.md

```diff
@@ -145,6 +144,29 @@ To use Vision Mode, add the `--vision` flag when starting the server:
 Vision Mode works best with the computer use models that are able to interact with elements using
 X Y coordinate space, based on the provided screenshot.
 
+### Connecting to existing Chrome instance via CDP
+
+These command line options are available:
+
+- `--cdp <url>`: Connect to an existing Chrome instance using CDP (Chrome DevTools Protocol).
+- `--reuse-session`: When using CDP, reuse the existing browser context and its first page instead of creating a new one
```

Comment:
Let's assume this is true by default and only implement reuse mode at this point.